templates/prescription/general_edit.html

```diff
@@ -115,9 +129,9 @@
     <div style='float:left;margin:5px 5px 5px 5px;'>
       {amcCollect amc_id='e_prescribe_amc' patient_id=$prescription->patient->id object_category='prescriptions' object_id=$prescription->id}
       {if not $amcCollectReturn}
-        <input type="checkbox" id="escribe_flag" name="escribe_flag">
+        <input type="checkbox" id="escribe_flag" name="escribe_flag" checked="checked">
```

Comment:
The thought was if the provider has weno enabled they need this checked. But later thinking about it. Not every prescription will be eRx. So this will be removed.